app/Filament/Clusters/ServiceManagement.php

```diff
@@ -50,9 +50,6 @@ class ServiceManagement extends Cluster
     // TODO Move into policy once created...
     public static function canAccess(): bool
     {
-        /** @var User $user */
-        $user = auth()->user();
-
-        return $user->hasAnyLicense([LicenseType::RetentionCrm, LicenseType::RecruitmentCrm]);
```

Comment:
Ever since we added the barry IDE support static analysis is able to understand the `auth()->user()` is the `User` Model.

I would prefer that we simplify it now that we can. What are your thoughts?

Reply:
When did we do that? Mine doesn't seem to understand it. I remember talking about it, but I never saw the change. Also `auth()->user()` can be either `User` or `SystemUser` so how would it know which?

Reply:
Ah okay, not sure why my machine has not problem with it. Maybe because I have the laravel IDE plugin. I will change this back